src/components/suggestions.ts

```diff
@@ -42,3 +56,21 @@ export const addSuggestion = async (
     [authorId, authorUsername, suggestion, state]
   );
 };
+
+export const getSuggestions = async (list: string | null): Promise<Suggestion[]> => {
+  const db = await openDB();
+  let res: Suggestion[];
+
+  if (!list) {
```

Comment:
If I'm not mistaken, list here is the state they are querying right? I think you and Charles have been talking about it, but it might be good to clarify. 

`list` here isn't semantically correct either. `state` or something else would make sense since we're querying `suggestionStates` already.

Reply:
yeah you prob should clean up all the `list` stuff into `state`

Reply:
oops i missed this file - thanks ppls :)